### ISSUE TITLE
MP4 and WEBM files from /apps/firstrunwizard/img/ Not Served

### ISSUE THREAD

**Issue by nooblag** [nextcloud/firstrunwizard#296]
### Steps to reproduce
1. Fresh install of Nextcloud 18.0.0 using nginx as specified in Nextcloud docs [here](https://docs.nextcloud.com/server/18/admin_manual/installation/nginx.html).
2. Create new user so 'firstrunwizard' appears

### Expected behaviour
firstrunwizard first page apparently tries to show MP4/WEBM video? i.e. /apps/firstrunwizard/img/Nextcloud.mp4

### Actual behaviour
/apps/firstrunwizard/img/Nextcloud.mp4 file is not served, the page area is blank, and when attempting to access /apps/firstrunwizard/img/Nextcloud.mp4 directly to check if file is being served, it 302 redirects to /.

### Server configuration

**Operating system**: Ubuntu 18.04.4 LTS

**Web server:** nginx 1.16.1

**Database:** psql (PostgreSQL) 10.12 (Ubuntu 10.12-0ubuntu0.18.04.1)

**PHP version:** PHP 7.2.24-0ubuntu0.18.04.3 (cli) (built: Feb 11 2020 15:55:52) ( NTS ) 

**Nextcloud version:** 18.0.0

**Where did you install Nextcloud from:**
nextcloud.com

**List of activated apps:**
Enabled:
  - accessibility: 1.4.0
  - activity: 2.11.0
  - admin_audit: 1.8.0
  - announcementcenter: 3.7.0
  - apporder: 0.9.0
  - bruteforcesettings: 1.5.0
  - calendar: 2.0.1
  - camerarawpreviews: 0.7.3
  - checksum: 0.4.4
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - contacts: 3.1.8
  - dav: 1.14.0
  - event_update_notification: 1.0.1
  - external: 3.5.0
  - extract: 1.2.3
  - federatedfilesharing: 1.8.0
  - files: 1.13.1
  - files_accesscontrol: 1.8.1
  - files_automatedtagging: 1.8.0
  - files_downloadactivity: 1.7.0
  - files_fulltextsearch: 1.4.1
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - files_videoplayer: 1.7.0
  - firstrunwizard: 2.7.0
  - groupfolders: 6.0.1
  - impersonate: 1.5.0
  - issuetemplate: 0.6.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - mail: 1.1.2
  - notes: 3.1.5
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - ocr: 6.0.16
  - onlyoffice: 4.1.4
  - password_policy: 1.8.0
  - photos: 1.0.0
  - previewgenerator: 2.2.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - quota_warning: 1.7.0
  - recommendations: 0.6.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - socialsharing_email: 2.0.0
  - spreed: 8.0.4
  - support: 1.1.0
  - suspicious_login: 3.1.0
  - systemtags: 1.8.0
  - tasks: 0.11.3
  - text: 2.0.0
  - theming: 1.9.0
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - user_retention: 1.1.0
  - user_usage_report: 1.2.0
  - video_converter: 0.1.3
  - viewer: 1.2.0
  - workflowengine: 2.0.0
Disabled:
  - encryption
  - federation
  - files_external
  - group_everyone
  - nextcloud_announcements
  - survey_client
  - terms_of_service
  - user_ldap

**Nextcloud configuration:**

```
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "skeletondirectory": "\/mnt\/ncdata\/__skeleton",
        "dbtype": "pgsql",
        "version": "18.0.0.10",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "htaccess.RewriteBase": "\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "log_type": "file",
        "logfile": "\/var\/log\/nextcloud\/nextcloud.log",
        "loglevel": "2",
        "mail_smtpmode": "smtp",
        "remember_login_cookie_lifetime": 0,
        "log_rotate_size": "10485760",
        "trashbin_retention_obligation": "auto, 14",
        "versions_retention_obligation": "auto, 365",
        "simpleSignUpLink.shown": "false",
        "filelocking.enabled": true,
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": "6379",
            "dbindex": "0",
            "timeout": "1.5"
        },
        "logtimezone": "Australia\/Melbourne",
        "maintenance": false,
        "default_locale": "en_AU",
        "force_locale": "en_AU",
        "default_language": "en",
        "force_language": "en",
        "defaultapp": "activity,files",
        "activity_expire_days": 180,
        "session_lifetime": 14400,
        "session_keepalive": false,
        "lost_password_link": "disabled",
        "overwriteprotocol": "https",
        "app_install_overwrite": [
            "activitylog",
            "uploaddetails",
            "drop_account"
        ]
    }
}
```

**Screenshots**

https://i.postimg.cc/W3SyXc7b/1.png
https://i.postimg.cc/281tFJLr/2.png

**Comment by nooblag** [comment]
I suspect this could have something to do with 'mp4' and 'webm' missing from this line at the end of the nginx conf (the same line that would catch png and jpg files)?

```
location ~ \.(?:png|html|ttf|ico|jpg|jpeg|bcmap)$ {
        try_files $uri /index.php$request_uri;
        access_log off;
    }
```

So consequently, it hits:

```
    location / {
        rewrite ^ /index.php;
    }
```

Maybe?

**Comment by juliushaertl** [comment]
Yes, this needs to be fixed by adjusting your nginx configuration as it is done in our documentation update https://github.com/nextcloud/documentation/commit/36f4ee1fce6bc73fa74a318498736ce2c03f0ef7#diff-490ed055ec68f0382edb4b4d680b761b

**Comment by figureone** [comment]
Just noting that the official docs for nginx config do not include serving mp4 and webm files statically: https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html#nextcloud-in-the-webroot-of-nginx

Perhaps the fix in [36f4ee1](https://github.com/nextcloud/documentation/commit/36f4ee1fce6bc73fa74a318498736ce2c03f0ef7#diff-490ed055ec68f0382edb4b4d680b761b) got lost along the way?
